frontend_lib/src/TracimTableLib/Columns/content/contentInformationColumn.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { createColumnHelper } from '@tanstack/react-table'
+
+import Icon from '../../../component/Icon/Icon.jsx'
+
+const contentInformationColumn = (settings, contentType, t) => {
+  const columnHelper = createColumnHelper()
+  return columnHelper.accessor(row => {
+    if (!row.content) return ''
+    const contentTypeInfo = contentType.find(info => info.slug === row.content.type) || { label: '' }
+    const statusInfo = contentTypeInfo.availableStatuses.find(
+      s => s.slug === row.content.statusSlug
+    ) || { label: '' }
+
+    return t(statusInfo.label)
+  }, {
+    header: () => (
+      <span>{settings.header}</span>
+    ),
+    id: 'information',
+    cell: props => {
+      if (!props.row.original.content) return null
+
+      const contentTypeInfo = contentType.find(info => info.slug === props.row.original.content.type) || { label: '' }
+      const statusInfo = contentTypeInfo.availableStatuses.find(
+        s => s.slug === props.row.original.content.statusSlug
+      ) || { label: '' }
+
+      return (
+        <div className='contentListItem__information'>
+          <span className='contentListItem__information__status'>
```

Comment:
Is this `span` element relevant?
I mean, you got a `div` then `span` that contains `icon` and `span`
Maybe you can remove the first `span`? Or am I missing something?

Reply:
IIRC this is as it somewhat was in the original "content information/status" component (i don't remember the specific file), minor modification could have been made but it is as close as the original component. Improving it could be good, but my goal for this PR is to stay as close as it was.